README.md

```diff
@@ -65,19 +65,10 @@ virtual environment.
 ```
 deactivate
 pip install awscli
-venv\scripts\activate
+<venv_name>\scripts\activate  # replace virtual environment name
```

Comment:
Since we have non-programmer users, they may be unfamiliar with the convention that "<...>" denotes a meta-term to be substituted, and there's a risk they'll just insert a name here but leave the angle-brackets. As such, I recommend using case will make the division of actual and meta text still fairly clear and the substitution to do more obvious (by mentioning it in the comment).

```suggestion
VENV\scripts\activate  # replace VENV with your virtual environment name
```